README.md

```diff
@@ -1,55 +1,70 @@
 # cfgov-refresh
 
-This repository contains the redesign-in-progress of consumerfinance.gov. This includes front-end assets and build tools, and configuration for [Sheer](https://github.com/cfpb/sheer) to load content from Wordpress and Django back-ends to elasticsearch to render the site.
+**Description**: This repository contains the redesign-in-progress of the [consumerfinance.gov](http://consumerfinance.gov) website. This includes front-end assets and build tools, and configuration for [Sheer](https://github.com/cfpb/sheer) to load content from WordPress and Django back-ends to Elasticsearch to render the site.
 
-### This project is a work in progress
+**Technology stack**:
```

Comment:
I think what's still rubbing me wrong is that OS X (and Homebrew) is not a requirement of _running_ the software. It's the way the core team contributes to it. I think it would behoove us to make it clear what the requirements for running are, and what the requirements (or most common path) for contributing are.

This comment thread is impossible to find if you close the page. Should we transfer this content somewhere?